acceptancetests/assess_deploy_webscale.py

```diff
@@ -83,6 +117,9 @@ def main(argv=None):
     with bs_manager.booted_context(args.upload_tools):
         client = bs_manager.client
         deploy_bundle(client, charm_bundle=args.charm_bundle)
+        raw_logs = extract_module_logs(client, module=args.logging_module)
+        timings = extract_txn_timings(raw_logs, module=args.logging_module)
```

Comment:
I would also think that the most useful thing would be something like:
import functools 
totalTime = functools.reduce(lambda x, y: x+y, map(float, timings))
I don't know if you want to use functools or not, but essentially convert from strings to floats, and then add them all up is likely to give you a better metric.
Maybe you want to roll that into the extract_* function?